src/main/java/com/deleats/service/MemberService.java

```diff
@@ -0,0 +1,42 @@
+package com.deleats.service;
+
+import com.deleats.dto.MemberDTO;
+import com.deleats.mapper.MemberMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class MemberService {
+
+    @Autowired
+    MemberMapper memberMapper;
+
+    /*
+    * ID 중복 체크
+    * */
+    public boolean duplicateId(String id){
+        boolean duplicate = true; //중복 체크 flag
+        int duplicateCnt = 0; //중복 체크
+
+        duplicateCnt = memberMapper.duplicateId(id); //Id 중복 체크
+
+        //Id 중복 여부에 따른 분기
+        if (duplicateCnt > 0){
+            duplicate = false;
+        }else {
+           duplicate = true;
+        }
+        return duplicate;
```

Comment:
이렇게 하면 한 줄로 줄일 수 있어보입니다~
```suggestion
        return duplicateCnt > 0 == false;
```

Reply:
오 감사합니다. 이런 방식은 뭐라고 불러야 할까요?